src/Events/EventsSent.ts

```diff
@@ -88,4 +96,12 @@ export default class EventsSent {
   public switchToProfile(profile: string, context: string, device: string): SwitchToProfileEvent {
     return new SwitchToProfileEvent(profile, context, device);
   }
+
+  public setFeedback(payload: LayoutFeedback, context: string) {
+    return new SetFeedbackEvent(payload, context);
+  }
+
+  public setFeedbackLayout(layout: LayoutFeedbackKey | string, context: string) {
```

Comment:
Yep. At present, `LayoutFeedbackKey` contains everything we know about that the system will accept. I put `string` there for the future, whenever Elgato allows us to create our own layouts.

If you'd rather I remove that so it can be done later, that's fine with me.

Reply:
yes, please remove then. maybe we'll never need it

Reply:
Done, although I worry that the typing in `SetFeedbackLayoutType` is just all sorts of cursed evil now. There _has_ to be a better way to do that one but it continues to elude me.

Reply:
great, thanks